docs/source/raft_configuration.md

```diff
@@ -13,16 +15,6 @@ documentation on [Planning for an ordering service](./deployorderer/ordererplan.
 
 ## Configuration
 
-While every Raft node must be added to the system channel, a node does not need
```

Comment:
Would it make any sense to replace this discussion with an overview of the new channel follower and consenter pattern?

Reply:
I think I'd rather wait on that until this doc is reconfigured/reorganized.